### PR TITLE
Product listing page

### PR DESCRIPTION
**Pull request by AswathyVNair**
Preview : https://deploy-preview-2--bakehere.netlify.app/pages/product-listing.html

### CODE REVIEW

**Review by SalonSikdar — APPROVED** [review]
Clean & Readable code, good to merge

File: css/product-listing.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
+.product-pg-container{
+    display: flex;
+}
+
+.pdoduct-pg-sidebar{
+    flex:1;
```
Comment (Muzammil-khan710) [inline css/product-listing.css:6]: what are these values?

File: css/product-listing.css [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    display: flex;
+    flex-direction: column;
+    border-right: 1px solid var(--light-grey);
+}
+
+.product-pg-maincontent{
+    padding: 1rem 0rem 1rem 2rem;
+    flex: 4;
```
Comment (Muzammil-khan710) [inline css/product-listing.css:14]: here also

Comment (AswathyVNair) [reply css/product-listing.css:14]: My sidebar and maincontent are two flex items. So I am mentioning their flex-growth.

**Pull request merged**